markdown/boomi-v2-getting-started/boomi-v2-getting-started.md

```diff
@@ -234,39 +214,44 @@ Positive
 
 ![canvas](img/canvas.png "canvas")
 
-### 2.. Configure Solace Shape
-👉 Hover over the Start shape to configure it 
+### 2. Search for the Solace shape from 
+
+![solace-shape](img/solace-shape.png "solace-shape")
 
-![canvas-start](img/canvas-start.png "canvas-start")
+👉 Drag and drop it into the canvas
 
-👉 Configure the Start shape to be a Solace PubSub+ Connector with the a **Listen** action as seen in the screenshot below
+👉 Alternatively, you can configure the Start shape to be a Solace PubSub+ Connector with the a **Listen** action as seen in the screenshot below
```

Comment:
I see it more as a sequence of steps rather than an alternative. It is also not quite clear that the Get is an option. 

1. Drag and drop it into the canvas
2. Then Select an Action for your Start shape - Get or Listen
3. ...

Not sure if we should briefly explain differences between Get and Listen?

Reply:
Right.. yea I think that would be a good idea to list all the different possible actions. Is there a place where the `Listen` `Send` and `Get` operations are documented? Or if you can provide me with one liners explanations for each and I'll add that as a note in the codelab

Reply:
https://help.boomi.com/bundle/connectors/page/int-Solace_PubSub_operation.html